### PR TITLE
Fill space stats

### PR DESCRIPTION
**Pull request by cfauci**
# Description
This is a tool so that I can compare gene assignment with fill space at scale. It allows me to output a success statistic and the beds of matching regions along with the distance between the bedpe regions that was associated with the successful match so I can compare that between runs

## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [x] I performed a self-review of my code
- [x] If it this a core feature, I have added thorough tests
- [x] The command `go fmt` or `make clean` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: bed/bedpe/compare.go [hunk 1]
```diff
@@ -26,3 +31,97 @@ func Equal(a BedPe, b BedPe) bool {
...
+			ChromEnd:          truth[currTruth].A.ChromEnd,
+			Name:              truth[currTruth].A.Name,
+			Annotation:        truth[currTruth].A.Annotation,
+			FieldsInitialized: 7}
+		truthAsBeds = append(truthAsBeds, trueBed)
+	}
+
+	mergedTruthBeds := bed.MergeBedsKeepNamesAndAnnotations(truthAsBeds)
+
+	for t := range mergedTruthBeds {
```
Comment (cfauci) [inline bed/bedpe/compare.go:64]: get rid of this check

File: bed/bedpe/compare.go [hunk 2]
```diff
@@ -26,3 +31,97 @@ func Equal(a BedPe, b BedPe) bool {
 	}
 	return true
 }
+
```
Comment (cfauci) [inline bed/bedpe/compare.go:34]: can be moved to a function because its so specific

**Review by RavenYL — APPROVED** [review]

**Review by edotau — APPROVED** [review]

**Review by edotau — DISMISSED** [review]

**Review by RavenYL — DISMISSED** [review]

**Review by RavenYL — APPROVED** [review]

**Review by hcnapier — APPROVED** [review]
